lib/response.ts

```diff
@@ -95,7 +95,7 @@ export default class Response {
         return this.header('Content-Type', mediaType);
     }
     unstate(name: string) {
-        cookie.delCookie(this as object, name);
+        cookie.deleteCookie(this as object, name);
```

Comment:
Thanks to https://github.com/denoland/deno/pull/6577, I believe the type assertion can be removed now, too.

This also applies to all of the cookie methods: `getCookie()`, `setCookie()`, and `deleteCookie()`.

Reply:
👍